tests/bytecode.rs

```diff
@@ -0,0 +1,65 @@
+use std::str::FromStr;
+
+use alloy_primitives::{bytes, Address, Bytes};
+use pevm::{Eip7702Code, EvmCode, LegacyCode};
+use revm::{
+    interpreter::analysis::to_analysed,
+    primitives::{Bytecode, Eip7702Bytecode},
+};
+
+#[test]
+fn test_evmcode_from_revm_bytecode_eip7702() {
+    let addr = Address::new([0x01; 20]);
+
+    // From address.
+    let bytecode = Bytecode::Eip7702(Eip7702Bytecode::new(addr));
+    let evmcode = EvmCode::from(bytecode);
+    assert!(
+        matches!(evmcode, EvmCode::Eip7702(Eip7702Code { delegated_address, .. })
+            if delegated_address == addr
+        )
+    );
+
+    // From raw.
+    let byte_str = format!("ef0100{}", addr.to_string().trim_start_matches("0x"));
+    let raw = Bytes::from_str(&byte_str).unwrap();
+    let bytecode = Bytecode::Eip7702(Eip7702Bytecode::new_raw(raw).unwrap());
+    let evmcode = EvmCode::from(bytecode);
+    assert!(
+        matches!(evmcode, EvmCode::Eip7702(Eip7702Code { delegated_address, version })
+            if delegated_address == addr && version == 0
+        )
+    );
+}
+
+#[test]
+fn test_evmcode_from_revm_bytecode_legacy_raw() {
```

Comment:
I'm in favour of the format: `test_<describe the test>` and it would be great to consistency across the test names. Please lmk what you think.

Reply:
I don't have a strong preference here. Feel the `test_` prefix is a bit redundant as there's usually a `#[test]` above the `fn`, and the log would be:
```
running 3 tests
test test_evmcode_from_revm_bytecode_eip7702 ... ok
test test_evmcode_from_revm_bytecode_legacy_analyzed ... ok
test test_evmcode_from_revm_bytecode_legacy_raw ... ok
```

Instead of just:
```
running 2 tests
test beneficiary_heavy_evaluation ... ok
test beneficiary_random ... ok
```

Reply:
No problems, will adjust.